pom.xml

```diff
@@ -173,17 +123,35 @@ under the License.
       <artifactId>icu4j</artifactId>
       <version>72.1</version>
     </dependency>
-
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.36</version>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>1.2.11</version>
+    </dependency>
```

Comment:
Why not SLF4J simple with runtime scope?

Reply:
fixed